src/chrome/content/rules/The-ACLU-of-Idaho.xml

```diff
@@ -0,0 +1,14 @@
+<ruleset name="The ACLU of Idaho">
+	<target host="acluidaho.org" />
+	<target host="www.acluidaho.org" />
+
+	<securecookie host="^acluidaho\.org$"
+			name=".+" />
+
+	<test url="http://acluidaho.org/" />
+	<test url="http://www.acluidaho.org/" />
+	<test url="https://www.acluidaho.org/" />
+
+	<rule from="^(http://(www\.)?|https://www\.)acluidaho\.org/"
```

Comment:
The preferred way to write this type of rule is like this:

```
<rule from="^http://www\.acluidaho\.org/"
  to="https://acluidaho.org />
<rule from="^http:"
  to="https:" />
```

Note that this version doesn't rewrite links from https://www.acluidaho.org/ to the no-www version. Since https://www.acluidaho.org/ doesn't work, it's very unlikely that anyone would link to it, so there's no need to fix it up. And we try to minimize rewrites of URLs that are already HTTPS wherever possible, to enable future performance optimizations.